tests/MDI_Test_Codes/driver_f90/driver_f90.f90

```diff
@@ -73,6 +73,10 @@ PROGRAM DRIVER_F90
    call MDI_Send_command("<NAME", comm, ierr)
    call MDI_Recv(message, MDI_NAME_LENGTH, MDI_CHAR, comm, ierr)
 
+   ! Use string to atomic number converter
+   call MDI_String_to_atomic_number("He", atomic_num, ierr)
+   WRITE(*,*) "Atomic Number for He: ", atomic_num
```

Comment:
The tests in `test_mdi.py` compare all lines output from the test drivers with an expected string, so if you write something here, you will need to edit the tests involving this test driver to expect this line.  Currently, the tests don't expect this line, so this causes them to fail.  I would suggest that you modify this to check if `atomic_num .ne. 2`, and only `write` something then.  That way, the tests will automatically fail if atomic_num is incorrect.